radix-engine/src/blueprints/package/package.rs

```diff
@@ -1493,16 +1154,16 @@ impl PackageRoyaltyNativeBlueprint {
         if royalty_charge.is_non_zero() {
             let handle = api.kernel_open_substate(
                 receiver,
-                MAIN_BASE_PARTITION,
-                &PackageField::Royalty.into(),
+                PackagePartition::Field.as_main_partition(),
```

Comment:
Again, would rather not rely on outside system code to specify what the main_partition is.